src/HotChocolate/Data/src/EntityFramework/Extensions/EntityFrameworkObjectTypeDescriptorExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using HotChocolate.Data;
+using HotChocolate.Data.DataLoaders;
+using Microsoft.EntityFrameworkCore;
+
+namespace HotChocolate.Types
+{
+    public static class EntityFrameworkObjectTypeDescriptorExtensions
+    {
+        public static IObjectFieldDescriptor Entity<TKey, TData, TProp, TDb>(
```

Comment:
It would be preferable if this method signature were merely `Entity<TData, TProp>`. The downstreams should have all the information they need to infer `TKey` and `TDb`. This will make the final syntax of this addition not require any explicit generic typing.

So instead of: `.Entity<string, Session, Speaker, MyDb>(s => s.speakers)`

The end-user code should just be: `.Entity(s => s.speakers)`